src/type-extensions.ts

```diff
@@ -29,5 +29,9 @@ declare module "hardhat/types/runtime" {
       init: Init
     ) => Promise<[Transaction, ScillaContract]>;
     zilliqa: ZilliqaHardhatObject;
+    getZilliqaChainId: () => number;
+    getNetworkUrl: () => string;
+    getPrivateKeys: () => string[];
+    setAccount: (num: number) => void;
```

Comment:
Does it set the active account? Maybe `setActiveAccount` or `setDefaultSigner` is a better name.

Reply:
Named after the function in `ScillaContractDeployer` that does this - perhaps we should rename that too? (there again, it is part of the API now). Renamed, anyway.